internal/install/stack_version.go

```diff
@@ -6,5 +6,5 @@ package install
 
 const (
 	// DefaultStackVersion is the default version of the stack
-	DefaultStackVersion = "8.0.0"
+	DefaultStackVersion = "8.0.1-SNAPSHOT"
```

Comment:
I had an offline chat with @jsoriano about this. Let's wait until we have the 8.0.1 release.